index.html

```diff
@@ -12,38 +12,77 @@
     <title>Atomist | Software delivery information for cloud-native applications</title>
   </head>
   <body>
-
+      
+<!-- Collapsed Navigation -->
+      
+<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">
+    <img src="src/assets/atomist_logo_shapes_banner-home.jpg" style="position:absolute;z-index:-10;left:-1500px;">
+    <img src="src/assets/Atomist-Logo-White-Horiz.png" style="width: 110px" class="m-3">
+    <div class="container h-100 ml-0 d-flex flex-column" style="z-index:10">
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Developer</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Pricing</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Docs</div>
+            <div class="nav-collapsed  row-1" style="z-idex:20;">Blog</div>
```

Comment:
I see that the style `z-index:20` is used several times, I'd suggest you put it as a class or variable in the css file.